api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.composite.v3;
+
+import "google/protobuf/any.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.composite.v3";
+option java_outer_classname = "CompositeProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Composite]
+// Composite Filter :ref:`configuration overview <config_http_filters_cors>`.
+// [#extension: envoy.filters.http.composite]
+
+// :ref:`Composite filter <config_http_filters_composite>` config. The composite filter config
+// allows delegating filter handling to an another filter as determined by a match result. This
+// makes it possible to use different filters or filter configurations based on the incoming request/response.
```

Comment:
I think you need to explain that this works as part of `ExtensionWithMatcher`. I.e. that's the entire meaning of this message basically.